src/py-opentimelineio/opentimelineio/url_utils.py

```diff
@@ -62,10 +65,18 @@ def filepath_from_url(urlstr):
     if PureWindowsPath(parsed_result.netloc).drive:
         filepath = Path(parsed_result.netloc + parsed_result.path)
 
-    # Otherwise check if the specified index is a windows drive, then offset the path
+    # Check if the specified index has a specified `drive`, if so then do nothing
+    elif filepath.drive:
+        filepath = filepath
+
+    # Check if the specified index is a windows drive, then offset the path
```

Comment:
I'm a bit concerned that this test only will work in a windows environment and that the "else" block below will manipulate almost all URLs passing through this function.

Reply:
Unfortunately, since I'm dev-ing on Windows I'm getting behavior that the CI can't replicate. Even the coverage results highlighted this section but it passes and is used on my machine.

Reply:
I removed this because your other suggestion from https://github.com/AcademySoftwareFoundation/OpenTimelineIO/pull/1674#discussion_r1375120236 is better.